x-pack/platform/plugins/shared/lens/common/content_management/v1/transforms/transforms.ts

```diff
@@ -17,15 +19,21 @@ import type { LensSavedObjectV0, LensAttributesV0 } from './types';
  * Includes:
  * - Legend value → Legend stats
  * - Stringified color mapping values → Raw color mappings values
+ * - Fix color mapping loop mode
+ * - Cleanup Lens SO attributes
+ * - Cleanup metric properties
  * - Add version property
  */
 export function transformToV1LensItemAttributes(
   attributes: LensAttributesV0 | LensAttributes
 ): LensAttributes {
-  return [convertToLegendStats, convertToRawColorMappingsFn, addVersion].reduce(
-    (newState, fn) => fn(newState),
-    attributes
-  );
+  return [
+    convertToLegendStats,
+    convertToRawColorMappingsFn,
+    attributesCleanup,
+    metricMigrations,
```

Comment:
shouldn't we also remove the existing metricMigrations in the metric code?

Reply:
No not until we have a way to convert by-value dashboard panels.